sdks/java/extensions/euphoria/euphoria-core/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/executor/io/FsSpillingListStorage.java

```diff
@@ -38,11 +39,12 @@
  * @param <T> the type of elements stored in this list storage
  */
 @Audience(Audience.Type.EXECUTOR)
+@SuppressFBWarnings("OBL_UNSATISFIED_OBLIGATION")
```

Comment:
is it because, its used only for testing?

Reply:
Nope. FindBugs compleins that streams are not closed propertly. Which is true. But fixing it would mean to redesign whole class or even the API. I would rather avoid thead in pert which will not be used in Beam in future.